src/test/java/tqs/example/impostor/repository/LockerRepositoryTest.java

```diff
@@ -0,0 +1,59 @@
+package tqs.example.impostor.repository;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.FactoryBasedNavigableListAssert.assertThat;
+
+@DataJpaTest
+public class LockerRepositoryTest {
+
+    @Autowired
+    private LockerRepository lockerRepository;
+
+    @Autowired
+    private TestEntityManager entityManager;
+
+    @Test
+    public void whenFindLockerByExistingId_thenReturnLocker() {
+        Locker locker = new Locker(null, "Locker 1", 10);
+        lockerRepository.save(locker);
+
+        Optional<Locker> foundLocker = lockerRepository.findById(locker.getId());
+
+        Assertions.assertThat(foundLocker).isPresent();
+        Assertions.assertThat(foundLocker.get().getId()).isEqualTo(locker.getId());
+        Assertions.assertThat(foundLocker.get().getAddress()).isEqualTo(locker.getAddress());
+        Assertions.assertThat(foundLocker.get().getCapacity()).isEqualTo(locker.getCapacity());
+    }
+
+    @Test
+    public void whenFindInvalidLockerId_thenReturnNull() {
```

Comment:
Although the naming in this case is quite clear, please consider giving it a name such as: "givenInvalidId_whenFindById_thenReturnEmpty".